core/src/main/java/com/bc/fiduceo/archive/Archive.java

```diff
@@ -69,7 +70,6 @@ static String[] createDefaultPathElements() {
     }
 
     // package access for testing only tb 2020-06-03
-
     static String[] relativeElements(Path archivePath, Path rootPath) {
```

Comment:
Why is this method static?
Only because of the tests or are there also concrete use cases in which this method is used without an archive instance?
I ask because the elements of the "rootPath" alternatif could also be created only 1 time, since the field "rootPath" is final.